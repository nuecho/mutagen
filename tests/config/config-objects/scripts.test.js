cfgObjectTests("scripts", "scripts-config.json");
