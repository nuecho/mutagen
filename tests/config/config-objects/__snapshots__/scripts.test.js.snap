// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen scripts config import-export] should create the scripts: the scripts have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ script => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen scripts config import-export] should update the scripts: the scripts have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ script => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
